### PR TITLE
Add http server helper document

### PR DESCRIPTION
**Pull request by ganmacs**
ref: https://github.com/fluent/fluentd/pull/2447

### CODE REVIEW

File: developer/api-plugin-helper-http_server.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Http Server Plugin Helper API
+
+`server_http` helper creates http server.
+
+Here is the code example with `server_http` helper:
```
Comment (repeatedly) [inline developer/api-plugin-helper-http_server.md]: `http_server`

File: developer/api-plugin-helper-http_server.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Http Server Plugin Helper API
+
+`server_http` helper creates http server.
```
Comment (repeatedly) [inline developer/api-plugin-helper-http_server.md]: `http_server`

File: developer/api-plugin-helper-http_server.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+### create\_http\_server(addr:, port:, logger:, default_app: nil, &block)
+
+This method creats and runs http server with given routes which are defined in `&block`.
+
+- `addr`: Adderess to listen to
+- `port`: Port to listen to
+- `logger`: Logger which is used in server helper
+- `default_app`: Use this object when server received a request whose path is not registered. This object must have `#call` or be a `Proc` object.
+
+** Define other HTTP methods
```
Comment (repeatedly) [inline developer/api-plugin-helper-http_server.md]: should mention return value and typical methods of `req`.

File: developer/api-plugin-helper-http_server.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+require 'fluent/plugin/input'
+
+module Fluent::Plugin
+  class ExampleInput < Input
+    Fluent::Plugin.register_output('example', self)
+
+    # 1. load http_server helper
+    helpers :http_server
+
+    config_param :host, :string
```
Comment (repeatedly) [inline developer/api-plugin-helper-http_server.md]: `bind`

**Comment by repeatedly** [comment]
Should add note that this helper is introduced since v1.6.0

File: developer/api-plugin-helper-http_server.md (hunk 1 above)
Comment (ganmacs) [reply developer/api-plugin-helper-http_server.md]: fixed https://github.com/fluent/fluentd-docs-gitbook/pull/58/commits/9c00cb19c5b3cfbd1447bc498c359e582d04228c

File: developer/api-plugin-helper-http_server.md (hunk 2 above)
Comment (ganmacs) [reply developer/api-plugin-helper-http_server.md]: fixed https://github.com/fluent/fluentd-docs-gitbook/pull/58/commits/9c00cb19c5b3cfbd1447bc498c359e582d04228c

File: developer/api-plugin-helper-http_server.md (hunk 3 above)
Comment (ganmacs) [reply developer/api-plugin-helper-http_server.md]: Added 
https://github.com/fluent/fluentd-docs-gitbook/pull/58/commits/593b04a304d6bab329dd58e3a80323a58de5b7e3

File: developer/api-plugin-helper-http_server.md (hunk 4 above)
Comment (ganmacs) [reply developer/api-plugin-helper-http_server.md]: https://github.com/fluent/fluentd-docs-gitbook/pull/58/commits/f1684570c1d866f58ee9b2cd0a2d5f2410cbec42

**Comment by repeatedly** [comment]
Thanks!

**Pull request merged**